### ISSUE TITLE
point_triangle_squared_distance function in point_triangle_squared_distances.cpp not being used at all.

### ISSUE THREAD

**Issue by velociraptor111** [alecjacobson/computer-graphics-bounding-volume-hierarchy#14]
It seems like function 'point_triangle_squared_distance' is not used anywhere in the code to produce correct results .. 
Also a closed issue mentioned once that this requirement was not on the README section part of the document, could it be that students don't need to use this function at all?

**Comment by ptanugra** [comment]
explained in  class

**Comment by velociraptor111** [comment]
@WilsonWangTHU  Yeah the prof mentioned in class that we don't necessarily have to use this function .. But we still have to implement it though and do our own testing as it will be graded
